### DISCUSSION TITLE
Combining 2 separate enclosure displays to act as one

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#84] [Ideas]
There are organs that have a swell pedal that controls a display on the console showing the level of activation of the pedal. In old organs this is all merchanical. I have created 2 enclosure element referencing the same manual and a set of bitmaps for each but both act independently. Can these be linked somehow?

![image](https://github.com/GrandOrgue/GoOdf/assets/6999990/3d86f351-c492-470d-85b6-ba6d1e6d51c3)

**Comment by larspalo** [comment]
@Bumblebee001 If you want the images to be linked to the same actions you're actually going the wrong way around doing this with multiple enclosures. Instead there should be **one** enclosure with _multiple GUI elements for the same_ enclosure (that each can be styled and positioned individually).

**Comment by Bumblebee001** [comment]
The elements are linked to the same enclosure.

**Reply by larspalo** [reply]
@Bumblebee001 If both GUI Elements really point to the same enclosure number, then what's your problem? Anyone of them can be used for interaction and any interaction will affect both of them.
